src/pages/storybook.js

```diff
@@ -335,11 +348,15 @@ export default () => (
                     <H4>Input (Narrow)</H4>
                     <InputStory narrow />
                     <br />
+                    <H4>TextArea</H4>
+                    <TextArea placeholder="Email Address" />
```

Comment:
This looks great! Do we also want to add it with the other `Form Elements` story outside of the modal?